crates/ui/src/components/keybinding.rs

```diff
@@ -16,31 +16,94 @@ impl RenderOnce for KeyBinding {
             .flex_none()
             .gap_2()
             .children(self.key_binding.keystrokes().iter().map(|keystroke| {
-                let key_icon = Self::icon_for_key(keystroke);
-
-                h_flex()
-                    .flex_none()
-                    .gap_0p5()
-                    .p_0p5()
-                    .rounded_sm()
-                    .text_color(cx.theme().colors().text_muted)
-                    .when(keystroke.modifiers.function, |el| el.child(Key::new("fn")))
-                    .when(keystroke.modifiers.control, |el| {
-                        el.child(KeyIcon::new(IconName::Control))
-                    })
-                    .when(keystroke.modifiers.alt, |el| {
-                        el.child(KeyIcon::new(IconName::Option))
-                    })
-                    .when(keystroke.modifiers.command, |el| {
-                        el.child(KeyIcon::new(IconName::Command))
-                    })
-                    .when(keystroke.modifiers.shift, |el| {
-                        el.child(KeyIcon::new(IconName::Shift))
-                    })
-                    .when_some(key_icon, |el, icon| el.child(KeyIcon::new(icon)))
-                    .when(key_icon.is_none(), |el| {
-                        el.child(Key::new(keystroke.key.to_uppercase().clone()))
-                    })
+                if cfg!(target_os = "macos") {
```

Comment:
I'm not a huge fan of doing conditional compilation like this within a UI component.

I think we need a different way of achieving this.

Reply:
Something like a get_key_hint(key_name) -> enum { KeyIcon, Text }?

Reply:
(messily) resolved by bff9321

Reply:
If anyone has suggestions for how to clean up this code/move the conditional compilation out of the component, I would appreciate it

Reply:
We'll be looking into how to do this in a more comprehensive way soon, the big concern is how do we test specialized windows / linux UI from our predominantly macOS dev team? That's a bit of a bigger question than this PR can handle on it's own. But once we've got a good approach, I'll let y'all know :)